libr/core/vmenus.c

```diff
@@ -2599,19 +2599,21 @@ R_API void r_core_seek_next(RCore *core, const char *type) {
 	ut64 next = UT64_MAX;
 	if (strstr (type, "opc")) {
 		RAnalOp aop;
-		if (r_anal_op (core->anal, &aop, core->offset, core->block, core->blocksize, R_ANAL_OP_MASK_BASIC)) {
```

Comment:
One more u missed it :)

Reply:
Nothing better to do on saturday morning than look at my fuckups, huh?

Reply:
It happens to everyone  np :) , the rest will be reviewd by @radare